euth/flashpoll/templates/euth_flashpoll/includes/project_form_result_flashpoll_tab.html

```diff
@@ -19,8 +19,14 @@ <h3 class="form-section-heading">{% trans 'Result '%}</h3>
                 <p class='form-help-text'>{{ form.project.result.help_text }}</p>
                 {% render_field form.project.result class="form-control" %}
                 <span class="help-block">{{ error }}</span>
-            </div>
-			{% if pollresult.noOfReceivedResults != 0 %}
+            </div>           
+            {% if poll %}
+                <div style="width: 39px;">
+                    <i class="fa fa-download fa-2x" tooltip="Download" tooltip-popup-delay="1000" tooltip-trigger="Download" tooltip-placement="left" onclick="exportResults({{pollresults|get_simplejson}}, {{poll.title|get_simplejson}})" tabindex="-1"></i>
```

Comment:
Please don't register onclick events like this, because we wan't to switch to [Content Security Policy](https://en.wikipedia.org/wiki/Content_Security_Policy) in the near future.